publify_core/app/models/archives_sidebar.rb

```diff
@@ -30,7 +30,8 @@ def parse_request(_contents, _params)
     article_counts = Content.find_by_sql(["select count(*) as count, #{date_func} from contents where type='Article' and published = ? and published_at < ? group by year,month order by year desc,month desc limit ? ", true, Time.now, count.to_i])
 
     @archives = article_counts.map do |entry|
-      month = (entry.month.to_i % 12) + 1
+      month = (entry.month.to_i % 12)
+      month = 12 if month == 0
```

Comment:
Write this in 1 line. Not ideal to redefine the same variable twice, one after each other.